.circleci/config.yml

```diff
@@ -34,6 +34,10 @@ jobs:
       - samvera/bundle:
           bundler_version: << parameters.bundler_version >>
           ruby_version: << parameters.ruby_version >>
+          # re-including ruby and rails versions in cache_version I believe
+          # works around what I think is a bug in samvera/bundle caching where
+          # it's re-using caches that are no good.
+          cache_version: 1-<< parameters.ruby_version >>-<< parameters.rails_version >>
```

Comment:
👎 I am very sorry to see that this was a major obstacle, I will create an issue in https://github.com/samvera/samvera-circleci-orb